inc/namespace.php

```diff
@@ -345,13 +345,18 @@ function set_comments_per_page( $value ) : int {
 /**
  * Ensure URLs do not contain any relative paths.
  *
- * @param string $url The dependency URL.
+ * @param string|null $url The dependency URL.
  * @param string $handle The dependency handle.
- * @return string
+ * @return string|null
  */
-function real_url_path( string $url, string $handle ) : string {
+function real_url_path( ?string $url, string $handle ) : ?string {
 	global $wp_scripts, $wp_styles;
-
+	
+	// Avoid odd behaviour if null or empty value is passed.
+	if ( empty( $url ) ) {
+		return $url;
```

Comment:
Would it make sense to log or trigger an error here with the $handle, to aid teams debugging this?

Reply:
I did think about it but it seems like this is just a convenient place to log the error rather the right place if that makes sense. WordPress itself should warn about invalid URLs provided for assets which we should fix upstream ideally.